workspace/apps/pidp/src/app/features/profile/pages/user-access-agreement/user-access-agreement.page.ts

```diff
@@ -125,7 +125,6 @@ export class UserAccessAgreementPage implements OnInit {
 
   public onBack(): void {
     this.navigateToRoot();
-    this.utilsService.scrollTop();
```

Comment:
We need this `scrollTop()` on the UAA page on mobile to counter this following behaviour. When on the UAA page, if you scroll down and read the agreement, and then click on the `back` button, you are navigated to the front page. However, because the UAA page view had scrolled way down, the view on the front page is also scrolled down.